climpred/checks.py

```diff
@@ -143,3 +143,21 @@ def match_initialized_vars(init, ref):
             f'got {init_vars} for init and {ref_vars} for ref.'
         )
     return True
+
+
+def has_valid_lead_units(xobj):
```

Comment:
Make sure to add a docstring here explaining what the function does.

Reply:
> also extracting the script when downloading SubX data from the server would be nice to have in a `util` or `tutorial` function for others to use (I'd be interested!)

Not sure what you mean by "extracting the script when downloading SubX data from the server".  The notebooks will show users examples of how to download SubX and NMME data from the server.  Its so simple with xr.open_dataset that no special script is needed. Its just two lines, (1) to construct the URL and (2) xr.open_dataset.   Anyway, let me know what you were referring to here and I'll see what I can do.